lua/tabline/icons.lua

```diff
@@ -4,6 +4,14 @@ local config = require('tabline.config')
 local utils = require('tabline.utils')
 
 -- Return icon with fg color
+M.get_left_separator = function(active, modified)
+    if modified == 1 then
+        return utils.get_item('TabLineModifiedSeparator', config.get('separator'), active)
+    else
```

Comment:
Remove redundant else here for consistency of how I have done it elsewhere

Reply:
Just removed. :smiley: